go-controller/extensions/pkg/node/node_windows.go

```diff
@@ -1,21 +1,54 @@
 package node
 
 import (
+	"encoding/json"
+	"net"
+	"os"
+
+	"github.com/cloudflare/cfssl/log"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/extensions/pkg/types"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kube"
 
 	kapi "k8s.io/api/core/v1"
 	"k8s.io/client-go/kubernetes"
+
+	"github.com/Microsoft/hcsshim/hcn"
+)
+
+const (
+
+	// Annotations consumed by this module (i.e. this module expects the following annotations to be already set)
+	// extensionDrMac is also set by InitSelf() on this node
+	extensionHostSubnet = "hybrid-sdn.extensions.ovn.kubernetes.io/hostsubnet" // Host subnet
+	extensionDrMac      = "hybrid-sdn.extensions.ovn.kubernetes.io/drmac"      // Distributed router/gateway MAC
+	extensionDrIp       = "hybrid-sdn.extensions.ovn.kubernetes.io/drip"       // Distributed router/gateway IP address
```

Comment:
@JocelynBerrendonner I added these constants (and the VNI) to extensions/pkg/types/types.go so we can share them

Reply:
Fixed in the latest iteration